x-pack/plugin/esql/qa/testFixtures/src/main/resources/aggregate_metric_double.csv

```diff
@@ -0,0 +1,2 @@
+some_field:aggregate_metric_double
```

Comment:
I put this here as a placeholder but I'm not quite sure how we would want to format the csv tests for this type.